src/ucs/debug/memtrack.c

```diff
@@ -52,12 +53,14 @@ static ucs_memtrack_context_t ucs_memtrack_context = {
 
 #ifdef ENABLE_STATS
 static ucs_stats_class_t ucs_memtrack_stats_class = {
-    .name = "memtrack",
-    .num_counters = UCS_MEMTRACK_STAT_LAST,
-    .counter_names = {
-        [UCS_MEMTRACK_STAT_ALLOCATION_COUNT] = "alloc_cnt",
-        [UCS_MEMTRACK_STAT_ALLOCATION_SIZE]  = "alloc_size"
-    }
+    .name          = "memtrack",
+    .num_counters  = UCS_MEMTRACK_STAT_LAST,
+    .class_id      = 0,
+    .counter_names =
+            {
+                    [UCS_MEMTRACK_STAT_ALLOCATION_COUNT] = "alloc_cnt",
+                    [UCS_MEMTRACK_STAT_ALLOCATION_SIZE]  = "alloc_size"
+            }
```

Comment:
pls revert this part to use previous indentation style (even though clang-format compains)